src/components/inputs/RadioButton/RadioButton.scss

```diff
@@ -0,0 +1,117 @@
+$C: #{$WEBEX_COMPONENTS_CLASS_PREFIX}-radio-button;
+
+.#{$C} {
+  display: flex;
+  position: relative;
+
+  &__title {
+    color: var(--wxc-input--title--normal);
+    @include body-fonts;
+    margin-left: 0.5rem;
+  }
+
+  &__background {
+    position: relative;
+    display: inline-block;
+    content: '';
+    cursor: pointer;
+    all: unset;
+    box-sizing: border-box;
+    border-radius: 50%;
+    min-width: 1rem;
+    width: 1rem;
+    height: 1rem;
+  }
+
+  &__foreground {
+    position: absolute;
+    display: inline-block;
+    content: '';
+    border-radius: 50%;
+    width: 0.375rem;
+    height: 0.375rem;
+    top: 0.3rem;
+    left: 0.3rem;
+  }
+}
+
+.#{$C}[aria-disabled='false'] {
```

Comment:
this can go under the previous block (the one that starts with .#{$C} and just add a & at the start of this selector:
```scss
  &[aria-disabled="false"] {